prow/jobs/custom/label-sync.yaml

```diff
@@ -36,25 +36,34 @@ presubmits:
         # Set --confirm=false to only validate the configuration file.
         - --confirm=false
         - --orgs=knative-sandbox,knative
-        - --github-app-id=$(GITHUB_APP_ID)
-        - --github-app-private-key-path=/etc/github/cert
-        - "--github-hourly-tokens=2000"
-        - "--github-allowed-burst=1000"
+        # Below had to be comment out when we moved labels.yaml to community.
+        #- --github-app-id=$(GITHUB_APP_ID)
+        #- --github-app-private-key-path=/etc/github/cert
+        #- "--github-hourly-tokens=2000"
+        #- "--github-allowed-burst=1000"
+        # We have to still use the GitHub oauth token for authentication, since
+        # label_sync does not allow using GitHub app token when having
+        # --confirm=false.
+        # This happens with peribolos as well. Might be an issue with prow?
+        - "--github-token-path=/etc/github/token"
+        - "--tokens=1200"
         - --debug
         volumeMounts:
-        - name: github-token
+        #- name: github-token
+        - name: oauth
           mountPath: /etc/github
           readOnly: true
-        env:
-        - name: GITHUB_APP_ID
-          valueFrom:
-            secretKeyRef:
-              name: github-token
-              key: appid
+        # env:
```

Comment:
I would suggest removing these instead of commenting them in yaml config for cosmetic reasons, unless you are planning to uncomment them soon. Mostly my personal preference, though.